docs/public/install-service-binding-operator.adoc

```diff
@@ -0,0 +1,42 @@
+== Installing Service Binding Operator
+
+This document walks you through the steps to install link:https://github.com/redhat-developer/service-binding-operator/tree/v0.3.0[Service Binding Operator v0.3.0] on OpenShift and Kubernetes clusters.
+
+=== Why do I need the Service Binding Operator?
+
+odo uses Service Binding Operator to provide the `odo link` feature which helps connect an odo component to a service or another component.
+
+=== Installing Service Binding Operator on OpenShift
+
+To install Service Binding Operator on OpenShift, refer link:https://www.youtube.com/watch?v=8QmewscQwHg[this video].
+
+=== Installing Service Binding Operator on Kubernetes
+
+Steps mentioned in this section were tested on link:https://minikube.sigs.k8s.io/[minikube]. We tested this on minikube v1.15.1 but it should work on v1.11.0 and above.
+
+For Kubernetes, Service Binding Operator is not yet available via the OLM. The team is link:https://github.com/redhat-developer/service-binding-operator/issues/727[working on making it available].
+
+To install the Operator, execute the following `kubectl` command:
+[source,sh]
+----
+$ kubectl apply -f https://gist.githubusercontent.com/dharmit/0e05be20e98c9271b2117acea7908cc2/raw/1e45fc89fc576e184e41fcc23e88d35f0e08a7e9/install.yaml
```

Comment:
could we move this in odo repo?

Reply:
We would actually get rid of it very soon once it's added into Service Binding Operator repo. https://kubernetes.slack.com/archives/C019LQYGC5C/p1606820080016700